resources/views/includes/elements/payment-options.blade.php

```diff
@@ -1,59 +1,92 @@
+@if (isset($variant) && $variant === 'the-marlay-group')
 <div class="container mt-4 p-5">
     <div class="card mt-4 col-md-12">
+        <div class="row mb-4 no-gutters">
+            <h3 class="lead p-heading p-heading--single">Become a Marlay Group Patron</h3>
+            <hr />
+            <div class="col-md-6 border d-flex rounded text-center p-container p-container--single">
+                <figure class="fluid">
+                    <img class="img-fluid p-image p-image--single" src="/images/payment-options/the-marlay-group.jpg"
+                        alt="person in museum" />
+                </figure>
+            </div>
+            <div class="col-md-6 d-flex flex-column justify-content-center">
+                <a href="https://fitz-tnew-test.tnhs.cloud/donate/contribute1?ct=1"
+                    class="btn btn-info btn-credit">Credit/Debit Card £1,500</a>
```

Comment:
Hi @ostafinskim ,

The pound sterling sign (£) should be implemented using the relevant HTML Entity code, which in this case is `&pound;`

For more information, please see:

https://en.wikipedia.org/wiki/List_of_XML_and_HTML_character_entity_references

And for an easy to read list of the different entity references available, see here:

https://css-tricks.com/snippets/html/glyphs/

If you could please make sure this is changed in the rest of the file as well :)

Reply:
All £ changed to`&pound;` for all project files.